apps/prairielearn/src/components/SideNav.html.ts

```diff
@@ -189,18 +189,14 @@ function CourseNav({
         <button
           type="button"
           class="btn dropdown-toggle dropdown-menu-right border border-gray bg-white w-100 d-flex justify-content-between align-items-center mb-2"
-          data-toggle="dropdown"
           aria-label="Change course"
           aria-haspopup="true"
           aria-expanded="false"
-          data-boundary="window"
-          ${!resLocals.authz_data.overrides
-            ? html`
-                hx-get="/pl/navbar/course/${resLocals.course.id}/switcher" hx-trigger="mouseover
-                once, focus once, show.bs.dropdown once delay:200ms"
-                hx-target="#sideNavCourseDropdownContent"
-              `
-            : ''}
+          data-bs-toggle="dropdown"
+          data-bs-boundary="window"
+          hx-get="/pl/navbar/course/${resLocals.course.id}/switcher"
+          hx-trigger="mouseover once, focus once, show.bs.dropdown once delay:200ms"
+          hx-target="#sideNavCourseDropdownContent"
```

Comment:
I was bothering me how the conditionally-rendered attributes weren't formatted or syntax-highlighted correctly. And as it turns out, this implementation actually caused a problem, as the dropdown would never be populated if overrides were on. I opted to move this behavior to the server so that we could unconditionally render these attributes.

Reply:
I made the same change to the classic navbar in 7d43822.